### PR TITLE
Add Reports Module and custom sql

### PR DESCRIPTION
**Pull request by shobanad16**

### CODE REVIEW

File: openmrs/i18n/home/locale_en.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "MODULE_LABEL_APOOINTMENTS_KEY": "Appointments",
     "MODULE_LABEL_PROGRAMS_KEY": "Programs",
     "MODULE_LABEL_APPOINTMENT_SCHEDULING_V2_KEY": "Appointment Scheduling",
-    "MODULE_LABEL_EXPORTS_KEY": "Exports",
-    "MODULE_LABEL_IMPLEMENTER_INTERFACE_KEY":"Implementer Interface"
+    "MODULE_LABEL_EXPORTS_KEY": "fgjfgjfExports",
```
Comment (supriyatw) [inline openmrs/i18n/home/locale_en.json]: There's a typo here @shobanad16

File: openmrs/apps/reports/extension.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (supriyatw) [inline openmrs/apps/reports/extension.json]: why is this added?

**Review by supriyatw — CHANGES_REQUESTED** [review]

File: openmrs/i18n/home/locale_en.json (hunk 1 above)
Comment (shobanad16) [reply openmrs/i18n/home/locale_en.json]: done

File: openmrs/apps/reports/extension.json (hunk 2 above)
Comment (shobanad16) [reply openmrs/apps/reports/extension.json]: not necessary. removed

Comment (tarunkumar-tw) [reply openmrs/apps/reports/extension.json]: @shobanad16 I still see the extension empty file. Please check if that empty file is needed or not and remove it

Comment (shobanad16) [reply openmrs/apps/reports/extension.json]: @tarunkumar-tw yes this file is needed. if this file is deleted, then i get error when i click reports module.

**Pull request merged**